scripts/verify-install.sh

```diff
@@ -132,20 +137,20 @@ for signature in $SIGNATURES; do
   echo ""
 
   # Then make sure that the hash of the installed binaries can be found in the
-  # signed list of hashes.
-  if ! grep -q "$LND_SUM" "$signature"; then
-    echo "ERROR: Hash $LND_SUM for lnd not found in $signature: "
+  # manifest.
+  if ! grep -q "$LND_SUM" "$MANIFEST"; then
```

Comment:
prbably out of scope for this PR, but could be useful to verify arbitrary lnd binary without having to install it.

Usecase is I build the release and verify the signatures without having to install it (I might have a dev build installed that I'm running)

Reply:
That's a good idea! Created a separate PR: https://github.com/lightningnetwork/lnd/pull/5021

Reply:
Is it possible to make the check stricter by grepping for `”^<hash>  “` so it is extracted from the first column?

Reply:
Yes, good idea. Done.